### ISSUE TITLE
geni-add-trusted-tool has stack trace on failed db authentication

### ISSUE THREAD

**Issue by tcmitchell** [GENI-NSF/geni-portal#505]
If `geni-add-trusted-tool` fails to authenticate to the database, a stack trace occurs:
```
$ geni-add-trusted-tool portal urn:publicid:IDN+myhost.example.com+authority+portal
Traceback (most recent call last):
  File "/usr/local/bin/geni-add-trusted-tool", line 76, in <module>
    sys.exit(main())
  File "/usr/local/bin/geni-add-trusted-tool", line 71, in main
    sys.stderr.write(e.pgerror)
TypeError: argument 1 must be string or read-only character buffer, not None
```

Reproduce by removing `$HOME/.pgpass`

 _Imported from trac ticket [#505](http://trac.gpolab.bbn.com/proto-ch/ticket/505),  created by tmitchel on 05-07-2013 at 16:06, last modified: 05-28-2013 at 15:01_

**Comment by tcmitchell** [comment]
This is fixed in development and will be part of a future release.

_Trac comment by tmitchel (github user: tcmitchell) on 05-07-2013 at 17:34_

**Comment by tcmitchell** [comment]
Part of #354. Resolves in 1.5.

_Trac comment by tmitchel (github user: tcmitchell) on 05-28-2013 at 14:51_
